code-studio/scripts/build-commands.js

```diff
@@ -86,9 +91,16 @@ exports.bundle = function (config) {
   }));
 
   // babelify tranforms jsx files for us
-  bundler.transform('babelify', {
-    compact: false
-  });
+  bundler.transform('babelify', { compact: false });
+
+  if (shouldMinify) {
```

Comment:
Would this var be more accurately named something like "prodBuild" at this point, as what it toggles is slightly more broad.